File: QViz.NET/Periscope.QViz/JSON/TestAPI.cs
using System;

namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for API Test Case Details
	/// </summary>
	public class TestAPI
	{
		/// <summary>
		/// Identifier of the Test API Object
		/// </summary>
		public string testAPIId;
		
		/// <summary>
		/// Identifier of the Test Case Object
		/// </summary>
		public string testCaseId;
		
		/// <summary>
		/// Identifier of the API Object
		/// </summary>
		public string apiId;
		
		/// <summary>
		/// Serial number of the Test API Object
		/// </summary>
		public int srNo;
		
		/// <summary>
		/// Date and Time on which this Model was Created
		/// </summary>
		public DateTime createdOn;
		
		/// <summary>
		/// Date and Time on which this Model was last Updated
		/// </summary>
		public DateTime updatedOn;
		
		/// <summary>
		/// API Object associated with this Test API
		/// </summary>
		public API api;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public TestAPI()
		{
			this.testAPIId = "";
			this.testCaseId = "";
			this.apiId = "";
			this.srNo = 0;
			this.createdOn = DateTime.Now;
			this.updatedOn = DateTime.Now;
			this.api = new API();
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/TestAPI.java
package com.periscope.qviz.json;

import java.util.Date;

/**
 * JSON Model for API Test Case Details
 */
public class TestAPI {

	/**
	 * Identifier of the Test API Object
	 */
	public String testAPIId;

	/**
	 * Identifier of the Test Case Object
	 */
	public String testCaseId;

	/**
	 * Identifier of the API Object
	 */
	public String apiId;

	/**
	 * Serial number of the Test API Object
	 */
	public Integer srNo;

	/**
	 * Date and Time on which this Model was Created
	 */
	public Date createdOn;

	/**
	 * Date and Time on which this Model was last Updated
	 */
	public Date updatedOn;

	/**
	 * API Object associated with this Test API
	 */
	public API api;

	/**
	 * Default Constructor
	 */
	public TestAPI() {
		this.testAPIId = "";
		this.testCaseId = "";
		this.apiId = "";
		this.srNo = 0;
		this.createdOn = null;
		this.updatedOn = null;
		this.api = new API();
	}

}
